src/gui/mzroll/mainwindow.cpp

```diff
@@ -2485,35 +2488,34 @@ int MainWindow::loadMetaCsvFile(string filename){
 
         }
 
-        qreal redF, greenF, blueF, alpha;
+        qreal redF = -1;
+        qreal greenF = -1;
+        qreal blueF = -1;
+        qreal alpha = -1;
         if (header.count("color") && header["color"] < N) {
 
             string colorName = fields[header["color"]];
-            redF = -1;
-            greenF = -1;
-            blueF = -1;
-            alpha = -1;
             QColor color;
             
-            if(colorName.empty()) 
-                continue;    
-            if (colorName[0] == '#') {
-                color.setNamedColor(QString::fromStdString(colorName));
-                color.getRgbF(&redF, &greenF, &blueF, &alpha);
-            } else {
-                try {
+            if(!colorName.empty()) {
```

Comment:
@sakshikukreja14 Any reason why the code from here on was modified in this patch? Once you've read the JSON from its correct path, I would assume there is no reason to make any further changes.

Reply:
Earlier, we were using "continue" instead. But it was skipping the implementation of assigning cohort values if color name was empty.